eng/Version.Details.xml

```diff
@@ -1,20 +1,20 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.NET.Sdk" Version="9.0.100-rtm.24512.1">
+    <Dependency Name="Microsoft.NET.Sdk" Version="9.0.100-rtm.24521.19">
       <Uri>https://github.com/dotnet/sdk</Uri>
-      <Sha>5b9d9d4677ea31d954533e9de2f95a3ea638135d</Sha>
+      <Sha>1efe47335726077cc75f52ec5b00ebf243f4c048</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="9.0.0-rtm.24510.3" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="9.0.0" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>226c0347b92c4f9649bcc7ad580f74cb0409580e</Sha>
+      <Sha>d3981726bc8b0e179db50301daf9f22d42393096</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-rtm.24510.3" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>226c0347b92c4f9649bcc7ad580f74cb0409580e</Sha>
+      <Sha>d3981726bc8b0e179db50301daf9f22d42393096</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.Current.Manifest-9.0.100.Transport" Version="9.0.0-rtm.24469.1" CoherentParentDependency="Microsoft.NETCore.App.Ref">
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.Current.Manifest-9.0.100.Transport" Version="9.0.0-rtm.24515.1" CoherentParentDependency="Microsoft.NETCore.App.Ref">
       <Uri>https://github.com/dotnet/emsdk</Uri>
-      <Sha>8e660ff41e91879977e3a9d837e068bd72234c26</Sha>
+      <Sha>8be5676af1ccf568b258133788a24aedd1a80994</Sha>
     </Dependency>
```

Comment:
@lewing @marcpopMSFT I'm trying to setup a subscription for our `release/9.0.1xx` branch going forward.

Should everything be working there? There are some emscripten packages that don't seem to exist:
```
error NU1102: Unable to find package Microsoft.NET.Workload.Emscripten.Current.Manifest-9.0.100 with version (= 9.0.0-rtm.24515.1)
```
But Maestro is able to track the coherent dependency.

Reply:
I think you need to flow the stable version as well if you want darc to automatically add the stable package feed. Basically, you're flowing the transport package (which has unstable branding and ended up on the dotnet9 feed) but using the actual package (which has stable branding and is on a unique feed)

https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-8be5676a/nuget/v3/index.json

Reply:
Ok, maybe this one instead, I see:

https://github.com/dotnet/runtime/blob/d5baa14e684167fabdd980ee16d307994398f912/eng/Version.Details.xml#L71-L74